README.md

```diff
@@ -69,31 +69,31 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
 
 #### Getting Started
 
-1. [Download this DMAPP]() and [deploy it to your DataMiner System](https://docs.dataminer.services/develop/TOOLS/TOOApplicationPackages/Installing_an_app_package.html). Installing this package will add the Automation scripts for Microsoft Teams in the Automation module along with some memory files that will be used by the Automation scripts to save the IDs of the created resources. Note that reinstalling this DMAPP will overwrite the Automation scripts and memory files with identical names!
+1. [Download this DMAPP]() and [deploy it to your DataMiner System](https://docs.dataminer.services/develop/TOOLS/TOOApplicationPackages/Installing_an_app_package.html). Installing this package will add the Automation scripts for Microsoft Teams in the Automation module along with some memory files that will be used by the Automation scripts to save the IDs of the created resources.
+
+   > Note that reinstalling this DMAPP will overwrite any Automation scripts and memory files with identical names.
 
 2. [Open DataMiner Cube](https://docs.dataminer.services/user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube.html) and open the Automation module (via apps > Automation). The added Automation scripts will be shown there. <details><summary>`show demo`</summary>![Gif-Automation](https://user-images.githubusercontent.com/109528797/186685478-9eac1cbf-f2d9-4c9a-8a6a-a2f499dbdcd9.gif)</details>
 
 #### Creating a Team
 
-1. Open the Automation module (via apps > Automation) and go to the '*automation scripts*' tab.
-
-2. Click the '*Create Team*' Automation script and click the '*Execute*' button.
+1. In the Automation module in DataMiner Cube, click the '*Create Team*' Automation script and click the '*Execute*' button.
 
-3. Fill in the necessary information, note **the fields are case sensitive**:
+2. Fill in the necessary information. Note that the **the fields are case sensitive**.
 
    - *Team Owner Email*: The email address of the owner of the team you are creating.
    - *Team Name*: The name of the team you are creating.
    - *Teams*: Select the memory file the script should use to save the ID of the team you are creating.
 
-4. Click the '*execute now*' button.  A team (with a General channel) will be created in Microsoft Teams with the [DataMiner bot](https://teams.microsoft.com/l/app/9a09d087-5d07-4481-b34f-cd053eab7925) already installed. <details><summary>`show demo`</summary>![Gif-CreateTeam](https://user-images.githubusercontent.com/109528797/186685886-ae5f1834-1c5c-438d-92e7-03740330e51d.gif)</details>
+3. Click the '*execute now*' button.  A team (with a General channel) will be created in Microsoft Teams, if the [DataMiner bot](https://teams.microsoft.com/l/app/9a09d087-5d07-4481-b34f-cd053eab7925) is installed. <details><summary>`show demo`</summary>![Gif-CreateTeam](https://user-images.githubusercontent.com/109528797/186685886-ae5f1834-1c5c-438d-92e7-03740330e51d.gif)</details>
```

Comment:
The bot doesn't create the team, our api (called from this automation script) is creating the team with the microsoft graph api, and is directly installing our bot also in the team. We do this so we can send the notifications using our bot.